lib/ja_serializer.ex

```diff
@@ -1,13 +1,30 @@
 defmodule JaSerializer do
   @moduledoc """
-  Provides a DSL for defining and how to serialize data to return in jsonapi.org 1.0 format.
+  Provides a behaviour and optional DSL for defining and how to serialize data.
 
-  See JaSerializer.Serializer for use.
+  The DSL can be opted out of by passing `dsl: false`, eg:
+
+      defmodule MyApp.PostView do
+        use JaSerializer, dsl: false
+      end
+
+  See JaSerializer.Serializer for details on the behaviour.
+
+  See JaSerializer.DSL for details on the DSL.
   """
 
-  defmacro __using__(_) do
-    quote do
-      use JaSerializer.Serializer
+  defmacro __using__(opts \\ []) do
+    # Default to using DSL for now.
+    opts = Keyword.put_new(opts, :dsl, true)
+    if opts[:dsl] do
+      quote  do
+        use JaSerializer.Serializer
+        use JaSerializer.DSL
+      end
+    else
+      quote do
+        use JaSerializer.Serializer
+      end
```

Comment:
looks like `JaSerializer.Serializer` is being used in both blocks.  What are your thoughts on move the `use` block outside of the if statement and only have a `use` statement for the dsl if it's requested?